src/plugins/intel_gpu/tests/test_cases/experimental_detectron_generate_proposals_single_image.cpp

```diff
@@ -0,0 +1,197 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
```

Comment:
Could you add TC that validates second output?

Reply:
As I see we are testing both outputs, aren't we? 
` EXPECT_NEAR(expected_roi_scores[i], roi_scores_ptr[i], 0.001) << "i=" << i`

and `EXPECT_NEAR(expected_rois[roi_idx], rois_ptr[roi_idx], 0.001) << "i=" << i << ", coord=" << coord;`